package.json

```diff
@@ -41,34 +41,34 @@
     "axios": "0.27.2",
     "chalk": "^4.0.0",
     "lodash": "^4.17.21",
-    "office-addin-manifest": "^1.6.2",
-    "office-addin-usage-data": "^1.3.2",
-    "open": "^6.0.0",
+    "office-addin-manifest": "^1.8.0",
+    "office-addin-usage-data": "^1.4.8",
+    "open": "^8.4.0",
     "rimraf": "^3.0.2",
     "unzipper": "^0.10.11",
     "uuid": "^8.3.2",
     "yeoman-generator": "^5.6.1",
     "yosay": "^2.0.2"
   },
   "devDependencies": {
-    "@types/lodash": "^4.14.150",
-    "@types/mocha": "^8.0.4",
-    "@types/node": "^14.0.1",
-    "@types/request": "^2.48.4",
-    "@types/unzipper": "^0.10.3",
-    "@types/yeoman-generator": "^5.2.8",
-    "@typescript-eslint/eslint-plugin": "^5.10.2",
-    "@typescript-eslint/parser": "^5.10.2",
-    "concurrently": "^7.0.0",
+    "@types/lodash": "^4.14.182",
+    "@types/mocha": "^9.1.1",
+    "@types/node": "^17.0.31",
+    "@types/request": "^2.48.8",
+    "@types/unzipper": "^0.10.5",
+    "@types/yeoman-generator": "^5.2.10",
+    "@typescript-eslint/eslint-plugin": "^5.22.0",
+    "@typescript-eslint/parser": "^5.22.0",
+    "concurrently": "^7.1.0",
     "copyfiles": "^2.4.1",
-    "eslint": "^8.8.0",
-    "eslint-config-prettier": "8.3.0",
-    "mocha": "^9.2.0",
-    "prettier": "^2.5.1",
-    "ts-node": "^9.0.0",
-    "typescript": "^3.9.2",
+    "eslint": "^8.14.0",
+    "eslint-config-prettier": "8.5.0",
+    "mocha": "^10.0.0",
+    "prettier": "^2.6.2",
+    "ts-node": "^10.7.0",
+    "typescript": "^4.6.4",
```

Comment:
Be careful about updating typescript versions.  I can't remember where it was, but once when I tried this it has some unintended consequences and I had to revert back to a previous version.

Reply:
It didn't seem to break anything in yo-office. I will double-check the tests but will process updating if they are all successful.